configuration/compatibility-testing/test-rp-msa-4.1.0-884.yml

```diff
@@ -0,0 +1,98 @@
+# Config for MSA 4.1.0-884 deployed to staging on paas
+
+server:
+  applicationConnectors:
+    - type: http
+      port: 8080
+  adminConnectors:
+    - type: http
+      port: 8081
+
+matchingServiceAdapter:
+  entityId: http://www.test-rp-ms-${INDEX}.gov.uk/SAML2/MD
+  externalUrl: https://test-rp-msa-staging-backcompat-${INDEX}.cloudapps.digital/matching-service/POST
+
+localMatchingService:
+  matchUrl: http://test-rp-staging-backcompat-${INDEX}.apps.internal:8080/test-rp/matching-service/POST
+  accountCreationUrl: http://test-rp-staging-backcompat-${INDEX}.apps.internal:8080/test-rp/unknown-user/POST
+  client:
+    timeout: 60s
+    timeToLive: 10m
+    connectionTimeout: 4s
+    tls:
+      verifyHostname: false
+      trustSelfSignedCertificates: true
+
+hub:
+  ssoUrl: https://www.staging.signin.service.gov.uk/SAML2/SSO
+  republishHubCertificatesInLocalMetadata: true
+  hubEntityId: https://signin.service.gov.uk
+
+metadata:
+  url: https://www.staging.signin.service.gov.uk/SAML2/metadata/federation
+  trustStore:
+    path: test_ida_metadata.ts
+    password: puppet
```

Comment:
Is a hard-coded, what looks like 'default' password here ok, instead of it being stored in a 'safe' somewhere?

Reply:
Java won't let you (elegantly) use a keystore or truststore without a password, even if it only contains public key material 😞

Reply:
Yeah this is a vexing limitation of java 😢 you have to use a password even if it's entirely non-secret content and doesn't need a password.